### ISSUE TITLE
Beetle Adventure Racing issue with Parallel graphics

### ISSUE THREAD

**Issue by Tasosgemah** [libretro/mupen64plus-libretro-nx#277]
At 1x resolution using Parallel it looks fine. But if i upscale the graphics a bit (2x) i get these artifacts in the 2nd level with the fog:

![Beetle Adventure Racing! (USA) (En,Fr,De)-201101-160525](https://user-images.githubusercontent.com/12124465/97805071-817db400-1c5c-11eb-9063-2a74c26aa5f5.png)

You can see the black pixels/artifacts around the trees in the background.

Everything else looks fine and i don't remember any other game having graphics issues with 2x resolution and Parallel graphics. This is the only issue i have found so far. Must have something to do with the game's fog, this particular game had issues with the fog in the past.

**Comment by m4xw** [comment]
I spoke to maister and its a wontfix unless there is proof this behaviour is a actual bug in parallel-rdp
> upscaling will have artifacts in some cases
> bilinear texture filtering tends to sample outside "expected" bounds
> Texture artifacts are expected when upscaling in some cases, and there's nothing I can do about it
